File: API/CreditsPythonApiDemo/keys.py
import base58


class Keys:
    def __init__(self, public_key, private_key, target_public_key):
        self.public_key = public_key
        self.private_key = private_key
        self.target_public_key = target_public_key
        self.public_key_bytes = base58.b58decode(public_key)
        self.private_key_bytes = base58.b58decode(private_key)
        self.target_public_key_bytes = base58.b58decode(target_public_key)

File: API/CreditsCSharpDemo/Keys.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CreditsCSAPIDemo
{
    public class Keys
    {
        public string PublicKey { get; set; }
        public string PrivateKey { get; set; }
        public string TargetKey { get; set; }
        public byte[] PublicKeyBytes
        {
            get
            {
                return PublicKey != null ? SimpleBase.Base58.Bitcoin.Decode(PublicKey).ToArray() : null;
            }
        }
        public byte[] PrivateKeyBytes
        {
            get
            {
                return PrivateKey != null ? SimpleBase.Base58.Bitcoin.Decode(PrivateKey).ToArray() : null;
            }
        }
        public byte[] TargetKeyBytes
        {
            get
            {
                return TargetKey != null ? SimpleBase.Base58.Bitcoin.Decode(TargetKey).ToArray() : null;
            }
        }
    }
}
